### DISCUSSION TITLE
bypass isConfirmed when logging in a user

### DISCUSSION THREAD

**Discussion by neverbe10** [strapi/strapi#7576] [Legacy Topics]
Hi, Strapi community:

first time posting question here. 
I wonder if there is a way to allow limited access for accounts that are not yet confirmed? As I understand it, when `/auth/local` is called, `permission.js` within `strapi-plugin-user-permissions` is called, and within that function, it checks if a user's `confirmed` field. When it's false, it throws an error, saying "Your account email is not confirmed."

But, in my app, I want to allow limited access for unconfirmed users. Is there a way to customize that logic?

**Comment by derrickmehaffy** [comment]
Hi @neverbe10 to customize the users-permissions plugin you will want to refer to this FAQ entry on the documentation for customizing how the plugin functions: https://strapi.io/documentation/v3.x/getting-started/troubleshooting.html#how-do-i-customize-a-plugin
